core/ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/workspace/FactoryWorkspaceComponent.java

```diff
@@ -210,12 +210,12 @@ public Boolean apply(UsersWorkspaceDto arg) throws FunctionException {
      * Create workspace with counter in name and add factoryId attribute
      * if workspace with specified name already exist.
      */
-    private Promise<UsersWorkspaceDto> createWorkspaceWithCounterName(final List<UsersWorkspaceDto> workspaces,
-                                                                      final WorkspaceConfigDto workspaceConfigDto) {
-        workspaceConfigDto.getAttributes().put(FACTORY_ID_ATTRIBUTE, factory.getId());
+    private Promise<WorkspaceDto> createWorkspaceWithCounterName(final List<WorkspaceDto> workspaces,
+                                                                 final WorkspaceConfigDto workspaceConfigDto) {
+//        workspaceConfigDto.getAttributes().put(FACTORY_ID_ATTRIBUTE, factory.getId());
```

Comment:
Yes i working on it, as attributes moved to workspace instance i figuring out what is the right place for that code